### DISCUSSION TITLE
Additional settings

### DISCUSSION THREAD

**Discussion by ukwuezeobinna** [modoboa/modoboa#2126] [General]
I have installed(Install Script) Modoboa and everything seems to work fine.

I need to know if there are additional setting that needs to be done? because I can't seem to add email accounts to email clients like Thunderbird. I also get messages that email is not encrypted.

Modoboa is hosted with AWS Lightsail and Port 25 is enabled. Are there additional ports that need to be enabled?

Please see screenshots.

Thank you.
![Screenshot_2021-01-15 Modoboa settings(2)](https://user-images.githubusercontent.com/852830/104717678-f58f0a80-5729-11eb-8261-fe21ed40d74c.png)
![Screenshot_2021-01-15 Modoboa settings(1)](https://user-images.githubusercontent.com/852830/104717722-050e5380-572a-11eb-84c9-037102b3a2b3.png)
![Screenshot_2021-01-15 Modoboa settings](https://user-images.githubusercontent.com/852830/104717726-06d81700-572a-11eb-9ad1-b66c9cbde75f.png)

**Comment by tonioo** [comment]
@ukwuezeobinna You need to open ports 143, 587, 4190 and 25 according to your settings.
